core/settings/base.py

```diff
@@ -284,8 +292,15 @@
         "name": "MIT License",
     },
 }
+
+# PASSWORD RESET
+# ------------------------------------------------------------------------------
+DJANGO_REST_RESET_TOKEN_EXPIRY_TIME = 10
+
 # EMAIL
 # ------------------------------------------------------------------------------
+CURRENT_DOMAIN = "https://ayushma.ohc.network"
```

Comment:
Can we have this configured through env?
```suggestion
CURRENT_DOMAIN = env("CURRENT_DOMAIN", default="http://localhost:8000")
```